src/hdmf_zarr/backend.py

```diff
@@ -94,7 +94,7 @@ def can_read(path):
              'default': None},
             {'name': 'object_codec_class', 'type': None,
              'doc': 'Set the numcodec object codec class to be used to encode objects.'
-                    'Use numcodecs.pickles.Pickle by default.',
```

Comment:
@magland just wondering if their may be some commits missing. I see that the comment in the docval string is changed but it looks like in line 124, ``self.__codec_cls = numcodecs.pickles.Pickle ...`` is still being set to Pickle in the PR.